src/EnrollCallAndSmsController.js

```diff
@@ -185,9 +191,11 @@ function (Okta, FormController, Footer, PhoneTextBox, TextBox, CountryUtil, Form
       var formSubmit = Okta.loc(isCall ? 'mfa.call' : 'mfa.sendCode', 'login');
       var formRetry = Okta.loc(isCall ? 'mfa.redial' : 'mfa.resendCode', 'login');
       var formSubmitted = Okta.loc(isCall ? 'mfa.calling' : 'mfa.sent', 'login');
-
+      
+      var warning = Okta.loc(isCall ? 'factor.call.time.warning' : 'factor.sms.time.warning', 'login');
       var numberFieldClassName = isCall ? 'enroll-call-phone' : 'enroll-sms-phone';
       var buttonClassName = isCall ? 'call-request-button' : 'sms-request-button';
+      var warningTimeout;
```

Comment:
you don't need this variable anymore, we should remove it

Reply:
fixed